### PR TITLE
krb5: krb5_msg_type rules

### PR DESCRIPTION
**Pull request by zer1t0**
Added rules that check the krb5_msg_type keyword. The pcap included contains kerberos traffic generated from a Windows server 2019 with the klist tool. It includes at least one packet to match per rule.

Related to the ticket https://redmine.openinfosecfoundation.org/issues/4529 and PR https://github.com/OISF/suricata/pull/6508

### CODE REVIEW

**Comment by zer1t0** [comment]
Done. This test is related to https://github.com/OISF/suricata/pull/6508.

File: tests/krb5-krb5_msg_type/test.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# *** Add configuration here ***
+requires:
+  features:
```
Comment (catenacyber) [inline tests/krb5-krb5_msg_type/test.yaml:3]: I think we will want min-version: 7

**Comment by catenacyber** [comment]
> The pcap included contains kerberos traffic generated from a Windows server 2019 with the klist tool

Could you include this in a REAMDE file in the test subdirectory ?

File: tests/krb5-krb5_msg_type/test.yaml (hunk 1 above)
Comment (zer1t0) [reply tests/krb5-krb5_msg_type/test.yaml:4]: Done in https://github.com/OISF/suricata-verify/pull/572/commits/1a993956803a8b72b42ad6631f9db78cc95c413b

**Comment by zer1t0** [comment]
> > The pcap included contains kerberos traffic generated from a Windows server 2019 with the klist tool
> 
> Could you include this in a REAMDE file in the test subdirectory ?

Done in https://github.com/OISF/suricata-verify/pull/572/commits/016460c422252b66603d092ba2a07cbe789c069c

**Comment by victorjulien** [comment]
Can you make a new version of this that passes on 5 and 6? I want to merge https://github.com/OISF/suricata/pull/6926 but not put SV in a broken state for 5/6. We can then update SV again as we consider backports of the issue. Thanks!

**Comment by zer1t0** [comment]
> Can you make a new version of this that passes on 5 and 6? I want to merge [OISF/suricata#6926](https://github.com/OISF/suricata/pull/6926) but not put SV in a broken state for 5/6. We can then update SV again as we consider backports of the issue. Thanks!

Done

**Comment by catenacyber** [comment]
Replaced by https://github.com/OISF/suricata-verify/pull/1194

**Pull request closed without merging**